src/components/selectKey/SelectKey.tsx

```diff
@@ -46,9 +46,9 @@ export default function SelectKey() {
   const hasFingerprints =
     publicKeyFingerprints && !!publicKeyFingerprints.length;
 
-  async function handleClick(fingerprint: Fingerprint) {
-    await dispatch(resetMnemonic());
-    await dispatch(login_action(fingerprint));
+  function handleClick(fingerprint: Fingerprint) {
```

Comment:
why did you remove async/await?

Reply:
Its not an async/await function so it isn't needed

Reply:
await does nothing in that case